man/systemd.service.xml

```diff
@@ -153,77 +153,89 @@
+            manager will delay starting of follow-up units until not only the service's main process has been created
+            but also the main service binary has been executed. (Or in other words: <option>simple</option> proceeds
+            with further jobs right after <function>fork()</function> returns, while <option>exec</option> will not
+            proceed before both <function>fork()</function> and <function>execve()</function> in the service process
+            succeeded.) Note that this means <command>systemctl start</command> command lines for
+            <option>simple</option> services will report success even if the service's binary cannot be invoked
+            successfully (for example because the selected <varname>User=</varname> doesn't exist, or the service
+            binary is missing), while the same command line for <option>exec</option> services will report failure in
+            such cases.</para></listitem>
+
+            <listitem><para>If set to <option>forking</option>, it is expected that the process configured with
+            <varname>ExecStart=</varname> will call <function>fork()</function> as part of its start-up. The parent
+            process is expected to exit when start-up is complete and all communication channels are set up. The child
+            continues to run as the main service process. This is the behavior of traditional UNIX services. If this
+            setting is used, it is recommended to also use the <varname>PIDFile=</varname> option, so that systemd can
+            identify the main process of the service. systemd will proceed with starting follow-up units as soon as the
+            parent process exits.</para></listitem>
+
+            <listitem><para>Behavior of <option>oneshot</option> is similar to <option>simple</option>; however, it is
+            expected that the process has to exit before systemd starts follow-up units.
+            <varname>RemainAfterExit=</varname> is particularly useful for this type of service. This is the implied
+            default if neither <varname>Type=</varname> nor <varname>ExecStart=</varname> are
+            specified.</para></listitem>
+
+            <listitem><para>Behavior of <option>dbus</option> is similar to <option>simple</option>; however, it is
+            expected that the service acquires a name on the D-Bus bus, as configured by
+            <varname>BusName=</varname>. systemd will proceed with starting follow-up units after the D-Bus bus name
+            has been acquired. Service units with this option configured implicitly gain dependencies on the
+            <filename>dbus.socket</filename> unit. This type is the default if <varname>BusName=</varname> is
+            specified.</para></listitem>
+
+            <listitem><para>Behavior of <option>notify</option> is similar to <option>simple</option>; however, it is
```

Comment:
Hmm, not that similar ;) It's closer to forking actually. Maybe say that this is another type where systemd is explicitly notified about the service being ready?

Reply:
Hmm, I'll change this to say it's like Type=exec actually, which I think is closer since Type=forking does this multiprocess jumble which we shouldn't reference here